src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/RazorPage/RazorPageGeneratorModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Web.CodeGeneration.CommandLine;
+
+namespace Microsoft.VisualStudio.Web.CodeGenerators.Mvc.Razor
+{
+    public class RazorPageGeneratorModel : CommonCommandLineModel
+    {
+        public string ViewName { get; set; }
```

Comment:
Might want to rename it to `RazorPageName` once we have the template content setup for RazorPages.